### ISSUE TITLE
bulk loading 0 triples causes an error

### ISSUE THREAD

**Issue by pchampin** [oxigraph/oxigraph#755]
**Describe the bug**
The bulk loader raises an error when asked to insert zero triples.

Granted, this is an edge case, but it feels like a bug. If only because: it is much more convenient for the methods of the bulk loader to detect that they received 0 quads, than for the caller of those methods to detect it in advance.

**To Reproduce**

1. Which version of Oxigraph are you using? On which platform?

github main (commit 86bbebf), on Ubuntu 23.10

2. A command line or a code snippet that triggers the bug.

```rust
    let store = oxigraph::store::Store::new()?;
    store.bulk_loader().load_dataset(b"".as_ref(), oxigraph::io::DatasetFormat::NQuads, None)?;
```
causes the following error:
```
Error: Storage(Other(ErrorStatus { code: 4, subcode: 0, severity: 0, message: "Invalid argument: ingestion arg list is empty" }))
```

**Comment by Tpt** [comment]
Thank you for this report. This issue seems to be a duplicate of #674 that has been fixed in [v0.3.21](https://github.com/oxigraph/oxigraph/releases/tag/v0.3.21) (later than the commit you refer to). Please reopen if it's still not fixed after updating your clone.

**Comment by pchampin** [comment]
My bad :-/ This is indeed resolved. Thanks
